src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -755,31 +775,32 @@ public void createEmptyInputScript_standard_redeemScript() {
     }
 
     @Test
-    public void createEmptyInputScript_fast_bridge_redeemScript() {
-        Script redeemScript = RedeemScriptUtils.createFastBridgeRedeemScript(
-            Sha256Hash.of(new byte[]{1}).getBytes(),
-            FEDERATION_KEYS
+    public void createEmptyInputScript_whenFlyoverStandardMultisig_ok() {
+        Script redeemScript = RedeemScriptUtils.createStandardRedeemScript(FEDERATION_KEYS);
+        Script flyoverRedeemScript = RedeemScriptUtils.createFlyoverRedeemScript(
+            FLYOVER_DERIVATION_HASH,
+            redeemScript
         );
 
-        Script spk = ScriptBuilder.createP2SHOutputScript(redeemScript);
-        Script inputScript = spk.createEmptyInputScript(null, redeemScript);
+        Script spk = ScriptBuilder.createP2SHOutputScript(flyoverRedeemScript);
```

Comment:
What spk mean?

Reply:
It is a legacy variable name. I updated the naming to what something clear.